package.json

```diff
@@ -60,7 +60,7 @@
         "gulp-sass": "^4.0.2",
         "husky": "^3.0.0",
         "jest": "^24.8.0",
-        "lerna": "^3.16.4",
+        "lerna": "^3.17.0",
```

Comment:
Please do upgrades like this in separate pr

Reply:
The reasoning for this is, unless the upgrade is nessesary to solve the task, this introduces extra risk in your PR. And can delay mergin the PR to deal with the extra risk. Handeling updates in the own PR makes it easier to revert if it fails in production. 

In Jøkul we have Greenkeeper, that might not work as it should at the moment, to upgrade dependecies and make PRs or issues if it fails or passes tests. Its a bit cleaner to let this be its own thing, istead of introducing it in a feature pr(unless you need some new function from a dependency)